nucypher/characters/control/specifications.py

```diff
@@ -47,9 +47,10 @@ class AliceSpecification(CharacterSpecification):
     __grant = (('bob_encrypting_key', 'bob_verifying_key', 'm', 'n', 'label', 'expiration'),  # In
                ('treasure_map', 'policy_encrypting_key', 'alice_verifying_key'))              # Out
 
-    # TODO: Implement Revoke Spec
-    __revoke = ((),  # In
-                ())  # Out
+    # A revoke specificatio isn't required because we don't use a serializer
+    # with it anyway.
+    __revoke = (('policy_encrypting_key'),  # In
```

Comment:
``` suggestion
    __revoke = (('policy_encrypting_key', ),  # In
```

Reply:
These will not be tuples without the commas, just a `str`!

Reply:
Ah, good catch.